app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
Custom exceptions should inherit from **_Exception_** class. BaseException is used for system exceptions.